tests/test_route53.py

```diff
@@ -326,3 +326,26 @@ def test_route53_domain_remove_tag(self):
         tags = client.list_tags_for_domain(
             DomainName=resources[0]['DomainName'])['TagList']
         self.assertEqual(len(tags), 0)
+
+class Route53EnableDNSQueryLoggingTest(BaseTest):
+
+    def test_hostedzone_querylogging(self):
+        session_factory = self.replay_flight_data('test_route53_enable_query_logging')
+        p = self.load_policy({
+            'name': 'enablednsquerylogging',
+            'resource': 'hostedzone',
+            'filters': [
+                {'Config.PrivateZone': False},
+                {'type': 'query-logging-enabled', 'state': False}],
+            'actions': [
+                {'type': 'enable-query-logging', 'state': True, 'logretentiondays': 60 }],
+                },
+            session_factory=session_factory)
+        self.assertEqual(len(p.run()), 1)
+        p = self.load_policy({
+            'name': 'query-logging-enabled',
+            'resource': 'hostedzone',
+            'filters': [{'type': 'query-logging-enabled', 'state': True}]},
+            session_factory=session_factory)
+        self.assertEqual(p.run()[0]['Id'], "/hostedzone/TESTABCXYZ1234")
```

Comment:
there isn't any validation that the query logging has been enabled here, just that the resource is filtered in.

Reply:
right, typically we'll construct a client and then do an api call (get_query_logging_config on the zone id) to verify the action's effects.